vocode/streaming/agent/token_utils.py

```diff
@@ -117,6 +119,12 @@ def get_tokenizer_info(model: str) -> Optional[TokenizerInfo]:
         )
         tokens_per_message = 3
         tokens_per_name = 1
+    elif "groq" == model: # TODO: Add appropriate tokenizer info for groq models
+        logger.debug(
```

Comment:
let's avoid token counting for groq at the moment, since the context limiting and token counting is openai-specific. So for now, delete this change